<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局字体设置为 Consolas */
* {
    font-family: Consolas;
}

/* 全局背景色设置为深灰色 */
QWidget, QFrame {
    background-color: #2b2b2b;
    color: #cccccc; /* 全局文本颜色设置为浅灰色 */
}

/* QComboBox 样式 */
QComboBox {
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 5px;
    background-color: #3c3f41;
    color: #cccccc;
    min-width: 75px;
}

QComboBox:hover {
    border: 1px solid #888888;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #555555;
    border-left-style: solid;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
    background-color: #3c3f41;
}

QComboBox::down-arrow {
    subcontrol-origin: margin;
    subcontrol-position: center center;
    width: 15px;
    height: 15px;
    image: none;
    color: #cccccc;
    font-size: 14px;
    text-align: center;
}

QComboBox::down-arrow::on {
    top: 1px;
    left: 1px;
}
        QComboBox:disabled {
            color: #808080; /* 禁用状态下文本颜色 */
            background-color: #404040; /* 禁用状态下背景颜色 */
            border-color: #404040; /* 禁用状态下边框颜色 */
        }
/* 下拉列表项样式 */
QComboBox QAbstractItemView {
    border: 1px solid #555555;
    selection-background-color: #007acc;
    selection-color: #ffffff;
    background-color: #3c3f41;
}

QComboBox QAbstractItemView::item {
    height: 25px;
    padding: 2px;
    color: #cccccc;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #4c5052;
    color: #ffffff;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #007acc;
    color: #ffffff;
}

/* QTreeWidget 样式 */
QTreeWidget {
    border: 1px solid #555555;
    background-color: #3c3f41;
    alternate-background-color: #2b2b2b;
    color: #cccccc;
}

QTreeWidget::item {
    height: 25px;
    padding: 2px;
}

QTreeWidget::item:selected {
    background-color: #007acc;
    color: white;
}

QTreeWidget::item:hover {
    background-color: #4c5052;
}

/* QHeaderView 样式 */
QHeaderView::section {
    background-color: #3c3f41;
    padding: 4px;
    border: 1px solid #555555;
    color: #cccccc;
}

/* QCheckBox 样式 */
QCheckBox {
    spacing: 5px;
    color: #cccccc;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:checked {
    content: &quot;✓&quot;;
    color: #007acc;
}

QCheckBox::indicator:unchecked {
    content: &quot;✗&quot;;
    color: #555555;
}

/* QPushButton 样式 */
QPushButton {
    background-color: #3c3f41;
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 5px;
    color: #cccccc;
}

QPushButton:hover {
    background-color: #4c5052;
}

QPushButton:pressed {
    background-color: #2b2b2b;
}

/* QLabel 样式 */
QLabel {
    color: #cccccc;
    background-color: #3c3f41;
    padding: 5px;
    border: 1px solid #555555;
    border-radius: 5px;
}

QLabel:hover {
    background-color: #4c5052;
}
        /* 自定义垂直滚动条 */
        QScrollBar:vertical {
            border: 2px solid #2b2b2b;
            background: #2b2b2b;
            width: 15px;
            margin: 20px 0 20px 0;
        }

        QScrollBar::handle:vertical {
            background: #007acc;
            min-height: 20px;
            border-radius: 7px;
        }

        QScrollBar::handle:vertical:hover {
            background: #5f5f5f;
        }

        QScrollBar::add-line:vertical {
            background: #2b2b2b;
            height: 20px;
            subcontrol-position: bottom;
            subcontrol-origin: margin;
        }

        QScrollBar::sub-line:vertical {
            background: #2b2b2b;
            height: 20px;
            subcontrol-position: top;
            subcontrol-origin: margin;
        }

        QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
            border: none;
            width: 1px;
            height: 1px;
            background: none;
        }

        QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
            background: none;
        }

        /* 自定义水平滚动条 */
        QScrollBar:horizontal {
            border: 2px solid #2b2b2b;
            background: #2b2b2b;
            height: 15px;
            margin: 0 20px 0 20px;
        }

        QScrollBar::handle:horizontal {
            background: #007acc;
            min-width: 20px;
            border-radius: 7px;
        }

        QScrollBar::handle:horizontal:hover {
            background: #5f5f5f;
        }

        QScrollBar::add-line:horizontal {
            background: #2b2b2b;
            width: 20px;
            subcontrol-position: right;
            subcontrol-origin: margin;
        }

        QScrollBar::sub-line:horizontal {
            background: #2b2b2b;
            width: 20px;
            subcontrol-position: left;
            subcontrol-origin: margin;
        }

        QScrollBar::left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
            border: none;
            width: 1px;
            height: 1px;
            background: none;
        }

        QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
            background: none;
        }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,1">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_3">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTreeWidget" name="treeWidget">
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Type</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>Mobile</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>HD_PC</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Level</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>LOD_0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_6</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_7</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LOD_8</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>292</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="10,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ExportPath</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
